webserver/src/main/java/io/kestra/webserver/services/ai/gemini/GeminiAiService.java

```diff
@@ -0,0 +1,362 @@
+                    ```
+                    {{flowSchema}}
+                    ```
+                    
+                    Whenever we're speaking about examples, properties or outputs in the below rules, they are strictly based on the above schema, any external source for those must be discarded because they are not a source of truth.
+                    Here are the rules:
+                    - If the user current flow seems unrelated, you can discard it and start from scratch, otherwise try to keep what you can from the current Flow while still replying to the user's intent.
+                    - Detect if the user wants an addition, deletion, modification or an entire rewrite of their Flow. If user mentions adding only part of the Flow (like adding a task for example), you should only add that part and not modify the rest of the Flow, even if the Flow contains some types that don't belong to the provided list.
+                      If nothing is mentioned regarding the change location or the scope, feel free to discard the initial Flow if it doesn't fit the user's need.
+                      If you can make any relation between what is existing and what the user's want, try to avoid duplicating intent (example, if a user say "say hello world" and it already has a Log task that says "hi", just replace the text "hi" with "hello world").
+                    - Use only the types and properties explicitly defined in the above schema. Do not invent, guess, or use properties from other types.
+                    - If a property is not present in the schema for a given type or you are unsure whether it exists or not DO NOT INCLUDE IT.
+                    - The type of each property must match the schema exactly.
+                    - Do not use any types not present in the schema in a given section.
+                    - When accessing variables within {{...}} expressions, you MUST ONLY use variables that are explicitly used in similar {{...}} expressions in the examples provided for that type or are explicitly documented as available for the specific task or context within the provided schema. Beware that some properties may not be available in the context of the task you are building.
+                    - There are a few global variables that are always available in the context of the Flow, such as `{{ flow.id }}`, `{{ flow.namespace }}`, `{{ flow.name }}`, `{{ flow.description }}`, `{{execution.id}}`, ... You can use them in your Pebble expressions but refer to kestra.io documentation to know if they exist.
+                    - Examples are very valuable and are fully working. Use them to guide property usage, structure, and Pebble expressions. However, don't copy them verbatim; adapt them to the current context, and if an example uses outputs, be careful about the type used in the example versus the one you are using.
```

Comment:
```suggestion
                    - Use provided examples to guide property usage and structure. Adapt them as needed; do not copy them verbatim.
```

This explains too much theory without giving clear instructions how to apply it. If you want to give more context, better show a specific example `"here is an example where user prompt <prompt> bla </prompt> can apply the example <example> text </example> to contruct the response <response>XYZ </response>"`